src/components/Menu.js

```diff
@@ -9,49 +12,133 @@ import { Link } from "react-router-dom";
 
 const Menu = ({ open, setOpen }) => {
   return (
-    <div
-      className={
-        "w-full " +
-        (open
-          ? "z-10 absolute w-full h-full bg-menu-transparent transition ease-in duration-300"
-          : "")
-      }
-    >
-      <div className="text-white mt-5 w-full sm:w-min px-8 lg:px-16">
-        <div className="inline-block sm:flex flex-row sm:space-x-8 w-full">
-          <div className="float-left">
-            <CgMenu
-              className="text-4xl md:text-5xl cursor-pointer"
-              onClick={() => setOpen(!open)}
-            />
-            {open ? (
-              <ul className="text-5xl text-white">
-                <li className="mt-4 hover:underline">
-                  <Link to="/about">About</Link>
-                </li>
-                <li className="mt-4 hover:underline">
-                  <Link to="/sponsors">Sponsors</Link>
-                </li>
-                <li className="mt-4 hover:underline">
-                  <Link to="/schedule">Schedule</Link>
-                </li>
-                <li className="mt-4 hover:underline">
-                  <Link to="/faq">FAQ</Link>
-                </li>
-                <li className="mt-4 hover:underline">
-                  <Link to="/register">Register</Link>
-                </li>
-              </ul>
-            ) : null}
-          </div>
-          <CgVolume
-            className={
-              "text-4xl md:text-5xl cursor-pointer float-right " +
-              (open ? "invisible" : "")
-            }
-          />
+    <div className="flex justify-center w-screen">
+      <div
+        className={
+          "text-white flex flex-row sm:space-x-8 w-full px-6 pt-6 mt-8 " +
+          (open ? "hidden" : "visible")
+        }
+      >
+        <CgMenu
+          className="text-4xl md:text-5xl cursor-pointer"
+          onClick={() => setOpen(!open)}
+        />
+        <div className="flex-1 sm:flex-none flex justify-end">
+          <RiVolumeUpFill className="text-4xl md:text-5xl cursor-pointer" />
         </div>
       </div>
+      <Transition show={open} as={Fragment}>
+        <Dialog
+          as="div"
+          id="modal"
+          className="fixed inset-0 z-10 overflow-y-auto"
+          static
+          open={open}
+          onClose={() => setOpen(false)}
+        >
+          <div className="min-h-screen">
+            <Transition.Child
+              as={Fragment}
+              enter="ease-out duration-300"
+              enterFrom="opacity-0"
+              enterTo="opacity-100"
+              leave="ease-in duration-700"
```

Comment:
The duration of the fade out should NOT be 700 ms. That is completely too long and unreasonable. I would try values between 200 and 300 ms and see which ones work best.

Reply:
lmao this wasn't intentional, my replace-all overwrote all the duration-200 lines! was planning to change this part back to how it was originally